### PR TITLE
#800 Improve code coverage in soul-client-springcloud

### PR DESCRIPTION
**Pull request by DaveModl**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mockStatic;
+
+/**
+ * Test Case for SpringCloudClientBeanPostProcessor.
+ *
+ * @author DaveModl (davemo-coderpersonal@hotmail.com)
+ */
+public final class SpringCloudClientBeanPostProcessorTest {
```
Comment (dengliming) [inline soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTest.java:49]: a new file? Do not overwrite the original file directly.

File: soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTestConfig.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import org.springframework.beans.factory.support.BeanDefinitionRegistry;
+import org.springframework.beans.factory.support.BeanDefinitionRegistryPostProcessor;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Test Case for SpringCloudClientBeanPostProcessor.
+ *
+ * @author DaveModl (davemo-coderpersonal@hotmail.com)
+ */
+public final class SpringCloudClientBeanPostProcessorTestConfig {
```
Comment (dengliming) [inline soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTestConfig.java:32]: Why we need this？

File: soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTest.java (hunk 1 above)
Comment (DaveModl) [reply soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTest.java:49]: ok

File: soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTestConfig.java (hunk 2 above)
Comment (DaveModl) [reply soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTestConfig.java:32]: init bean SpringCloudClientBeanPostProcessor

Comment (dengliming) [reply soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTestConfig.java:32]: In this case. We only need to test `SpringCloudClientBeanPostProcessor`. There is no need to verify Spring's bean initialization. You can just new/mock  one instance.

Comment (dengliming) [reply soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTestConfig.java:32]: In this case. We only need to test SpringCloudClientBeanPostProcessor. There is no need to verify Spring's bean initialization. You can just new/mock one instance.

File: soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTest.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        ENVIRONMENT.setProperty("spring.application.name", "testApp");
+
+    }
+
+    @Test
+    public void test() {
+        SpringCloudClientBeanPostProcessor origin = new SpringCloudClientBeanPostProcessor(CONFIG, ENVIRONMENT);
+        Assert.assertNotNull(origin);
+        //full
+        Assert.assertEquals(origin, origin.postProcessAfterInitialization(origin, "springCloudClientBeanPostProcessor"));
```
Comment (tydhot) [inline soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTest.java:73]: why not use a new Object?In spring container, BeanPostProcessor will not be a parameter in that function 'postProcessAfterInitialization'

**Comment by dengliming** [comment]
Since it hasn't been updated for a long time. we closing it. Feel free to reopen a new PR for this.

**Pull request closed without merging**